app/views/shared/_checkbox_group.html.haml

```diff
@@ -1,13 +1,15 @@
 -# for options see https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#govuk_collection_check_boxes-instance_method
-- if search
-  .search-input
-    %input.govuk-input.search-input__input{ placeholder: "Search" }
-    %button{type: "submit"}
-      %i.fa.fa-search
-.govuk-form-group
-  .govuk-checkboxes.govuk-checkboxes--small
-    = form_for '', url: 'url', html: { method: 'post' } do |f|
-      = f.govuk_collection_check_boxes :options,
-        options,
-        :last,
-        :first
+.tv-checkbox__group
+  - if search
+    .search-input
+      %input.govuk-input.search-input__input{ placeholder: "Search" }
+      %button{type: "submit"}
+        %i.fa.fa-search
+  .govuk-form-group
+    .govuk-checkboxes.govuk-checkboxes--small
+      = form_for '', url: 'url', html: { method: 'post' } do |f|
```

Comment:
And here we just have...

`= f.govuk_collection_check_boxes attribute_name,
     options,
     value_method,
     text_method,
     hint_method`